### ISSUE TITLE
dev network node state not running

### ISSUE THREAD

**Issue by ppca** [near/mpc-recovery#592]
debugging what happened

**Comment by ppca** [comment]
seems like node state is not advancing successfully from Starting to Started. 
Investigating why

**Comment by ppca** [comment]
seems like it has been retrying load triples 3 times and silently failing. It went into running state after I deleted all triples from dev storage. Will get in a fix for load triples retry next.

**Comment by ppca** [comment]
The only possible caveat I could find is that we changed the key when we do entity into_value() in this [PR](https://github.com/near/mpc-recovery/commit/d24cae0879665b41a0c96601162afd0217de6522#diff-46fbfe8c79db78157047bf2d1f1427e104c440d83858af99f99b130a81ce9b8e), where we used to use key: "triple", but now it's key:  None. 
We might potentially have some schema incompatibility issues, but I'm not sure if that explains why the load_triples() retried without printing success or failure messages.
Since both dev and testnet is working well at the moment, let's leave it as is, since all the data in the datastore should be compatible with the current code now.
If this happens again, we will investigate other causes.
